js-exercises/negative-index/negativeIndex.js

```diff
@@ -0,0 +1,35 @@
+function isNumber(numberValue) {
+  if (!numberValue || typeof numberValue === "symbol") {
+    return false;
+  }
+  numberValue = Number(numberValue);
+  return typeof numberValue === "number" && !isNaN(numberValue);
+}
+
+function negativeIndex(array) {
+  if (!Array.isArray(array)) {
+    throw TypeError("Only arrays are supported.");
+  }
+
+  const handlers = {
+    get(target, property, receiver) {
+      let getThisIndex = property;
```

Comment:
I did not find this variable name suitable! `getThisIndex` feels like we are returning the index but we are returning the value at that index. So it will be better if you try renaming it around the `index` property